aws/app/lambda/submission/submission.js

```diff
@@ -65,6 +66,7 @@ const saveData = async (submissionID, formData) => {
       SendReceipt: { S: "unknown" },
       FormSubmissionLanguage: {S: formData.language},
       FormData: { S: formSubmission },
+      SubmissionDateTime: {S: new Date().getTime()}
```

Comment:
I would change it for `SubmissionTimestamp` to better reflect what the value is. It should also be a type `N` for number instead of a string.